### ISSUE TITLE
does this run on GD32F303?

### ISSUE THREAD

**Issue by joker-mik** [mean00/lnDSO150#2]
Does this Version also run on GD32F303 like DSO_STM32Duino?
This is not mentioned in Wiki.

**Comment by mean00** [comment]
Yes, STM32F103/GD32F103/GD32F303/GD32VF103 and hopefully soon CH32F103
 i've updated the (in-progress) wiki
Thank you
